src/app/entries/entry-list/entry-forecast-list.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { ApiService } from '../../services/api.service';
+import { Entry } from '../../shared/entry'
+
+@Component({
+  selector: 'app-entry-forecast-list',
+  templateUrl: './entry-forecast-list.component.html',
+  styleUrls: ['./entry-forecast-list.component.scss']
+})
+
+export class EntryForecastListComponent implements OnInit {
+  public entrylist: Entry[];
+  public daysofWeek = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'];
+
+  constructor(private  apiService:  ApiService) { }
+
+  ngOnInit() {
+    this.getPosts();
+  }
+
+  getPosts() {
+    this.apiService.getEntries().subscribe((data) => {
```

Comment:
Eventually we will want to preload this data before the view loads so that the page doesn't flicker, no need to worry about this for now though.